client/jest.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     // this number should only be getting closer to 0).
     global: {
       lines: -161,
-      branches: -194,
+      branches: -195,
```

Comment:
Looking at the detailed coverage report, I couldn't find a single uncovered branch in any lines included in this diff.

Reply:
This has been happening to me lately as well. Can't figure it out!